Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderContainer.js

```diff
@@ -1,2 +1,21 @@
 import React from 'react';
 import './Header.css';
+
+const HeaderContainer = () => {
+    return (
+      <div className = "container">
+        <div>
+          <img src="https://ibin.co/3whrpKSBbZ81.png" alt = "" className = "headerImg"/>
+        </div>
+        <div className = "headerText">
+          <div className = "headerTitle">
```

Comment:
Depending on how you want your components structured, you can either abstract both HeaderTitle and HeaderContent out here OR abstract out HeaderContent here and further abstract out HeaderTitle from within HeaderContent. I think it makes the most sense to choose the former.